examples/websocket-relay/README.md

```diff
@@ -6,27 +6,30 @@ An example application that connects to a `homestar-runtime` node
 over a websocket connection in order to run static Wasm-based, image
 processing workflows that chain inputs and outputs using
 [inlined promises][pipelines].
-
+ 
 This application demonstrates:
 
-  * websocket notifications of [Ucan Invocation receipts][spec-receipts] sent
+  * websocket notifications of [UCAN Invocation Receipts][spec-receipts] sent
     between a web client and a `homestar` runner
   * instantaneous replay of previously run, cached executions
   * fetching content (the original static image) over [IPFS][ipfs]
     through a local blockstore
-  * the [WIT][wit] + [Ipld][ipld] interpreter for
+  * the [WIT][wit] + [IPLD][ipld] interpreter for
     [Wasm(time)][wasmtime] embedded execution within a `homestar` runner.
 
 ## Install
 
-To get started, please install:
+### Nix
+
+If you're using our [Nix file](../../flake.nix), you get these installs for free.
 
-* [rust][install-rust], unless you're running `homestar` [as a binary][rust-binary]
-* [node & npm][install-npm]
-* [kubo/ipfs installed][install-ipfs]
+### Manual
+
+To get started, please install:
 
-If you're using our [nix file](../../flake.nix)], you get these installs for
-free.
+* [Rust][install-rust], unless you're running `homestar` [as a binary][rust-binary]
```

Comment:
This extra `]` is what brought me here initially:

<img width="255" alt="Screenshot 2023-10-04 at 18 11 48" src="https://github.com/ipvm-wg/homestar/assets/1052016/f98fbbf3-300b-4dfb-960b-a48dd29ca7b4">